### ISSUE TITLE
PayloadCMS integration

### ISSUE THREAD

**Issue by ixahmedxi** [ixahmedxi/orbitkit#8]

**Comment by imCorfitz** [comment]
Looking much forward to this one. You planning on waiting till we see a release of the 3.0 Next.js based CMS?

**Comment by ixahmedxi** [comment]
Yes I think we will wait for the 3.0 release, however I'm not sure how that will look like because we can't have it in `apps/web` because that's really the product's app and `apps/marketing` is in astro so it won't be able to be the source of the payload cms instance. creating a new nextjs app for it solely as well feels like a waste so we will need to assess the situation when payload 3.0 is out.

**Comment by imCorfitz** [comment]
I would argue that PayloadCMS today is a headless CMS and therefore it wouldn't be wrong to create a dedicated `apps/cms` app for the Payload installation. Granted version 3.0 should allow you to easily deploy an app and admin in a single deployment, however I have used payload as a headless CMS for many applications and can't see that go away anytime soon.

**Comment by ixahmedxi** [comment]
Yeah I agree here too, I think a dedicated `apps/cms` makes a lot of sense.
